package.json

```diff
@@ -4,45 +4,43 @@
   "description": "Nest TypeScript starter repository",
   "license": "MIT",
   "scripts": {
-    "build": "tsc -p tsconfig.build.json",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
```

Comment:
Why remove?

Reply:
`format` and `build` have not been removed. I just moved `format` and `lint` to the bottom of the scripts list and everyhting related to `start` the app to the top